### ISSUE TITLE
Mixing test types?

### ISSUE THREAD

**Issue by jorbor** [olgakl/OlgasInventorySystem#2]
Controller tests are using faked repo and product repo tests are making HTTP calls. Do you feel like these are different types of tests, and if so, 1. how would you label them/differentiate them, 2. should they be in different projects?

**Comment by olgakl** [comment]
Yes, these are different types of tests. The one that tests the data layer
is an integration test. I wanted to verify that my data access layer works
properly, but yes, these do belong in a different test project.
